specialized/specialized_form_parser.ipynb

```diff
@@ -101,12 +101,8 @@
     "    document = result.document\n",
     "    entities = document.entities\n",
     "    print(\"Document processing complete.\\n\\n\")\n",
-    "    \n",
```

Comment:
Why did we remove this but keep the rest of the HITL code in the bottom?

Reply:
How very strange, if you look at the full diff of my most recent commit it's in there